src/ops/ops.go

```diff
@@ -165,15 +187,24 @@ func (o *ops) WriteImageToExistingRoot(liveLogger io.Writer, ignitionPath string
 	}
 	o.log.Infof("imported commit %s", commit)
 
-	out, err = o.ExecPrivilegeCommand(liveLogger, "ostree", "admin", "deploy",
-		"--stateroot", "install",
-		"--karg", "$ignition_firstboot",
-		"--karg", defaultIgnitionPlatformId,
-		commit)
+	out, err = o.ExecPrivilegeCommand(liveLogger, "ostree", ostreeArgs(commit, installerArgs)...)
 	if err != nil {
 		return errors.Wrapf(err, "failed to deploy commit to stateroot: %s", out)
 	}
 
+	if slices.Contains(installerArgs, "--copy-network") || slices.Contains(installerArgs, "-n") {
+		o.log.Info("copying network files from /etc/NetworkManager/system-connections to /boot/coreos-firstboot-network")
+		out, err = o.ExecPrivilegeCommand(liveLogger, "mkdir", "/boot/coreos-firstboot-network")
+		if err != nil {
+			return errors.Wrapf(err, "failed to create firstboot network directory: %s", out)
+		}
+
+		out, err = o.ExecPrivilegeCommand(liveLogger, "sh", "-c", "cp /etc/NetworkManager/system-connections/* /boot/coreos-firstboot-network")
```

Comment:
FWIW this was done this way because without the additional shell the glob was getting evaluated in the container, then the command failed when run on the host. I had to do it this way to ensure the glob was evaluated on the host.

I can add a comment if folks think it's worth it.